examples/getting_started/react/components/Todo/Index.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import Content from './TodoContent';
```

Comment:
This is one of the potential drawbacks of default exports. We can basically assign arbitrary names to whatever was actually exported from the module. Sometimes, this makes things a little less obvious.

It might be a little bit clearer if we named this `TodoContent`